app.py

```diff
@@ -95,6 +103,31 @@ def most_similar_by_embedding(
         request.threshold,
         include_scores,
     )
+
+    if request.save_question:
```

Comment:
i think this should default to True so we store the data without changing anything in cognition.